scripts/ci-cron.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 
+set -o nounset
 set -o errexit
 
-for package in $(cat sources.txt) doc/tutorial ; do
+export PATH=$(stack path --bin-path):$PATH
+
+stack install cabal-install
+cabal update
```

Comment:
The `install` section of the `.travis.yml` also gets executed for cron jobs. (That's where `stack` is installed: https://github.com/haskell-servant/servant/pull/568/files#diff-354f30a63fb0907d4ad57269548329e3R18 .) So when executing `ci-cron.sh` on travis we already have `stack`. But no `ghc` and no `cabal-install`. So I thought it would be easiest to just use `stack`'s `ghc` and use it to install `cabal-install`.

Alternatively we could install `ghc` and `cabal-install` from hvr's repos, like before. That would have the disadvantage of slowing down the push-triggered builds unnecessarily.

Generally the `ci-cron.sh` script is not really how I want it to be. But I think it should be worked on in follow-up PRs. So that we can get the stack CI in place. (And I think it's already better than what we have now.)